addons/web/static/src/legacy/js/views/list/list_renderer.js

```diff
@@ -209,7 +209,8 @@ var ListRenderer = BasicRenderer.extend({
         const decorations = {};
         for (const [key, expr] of Object.entries(node.attrs)) {
             if (DECORATIONS.includes(key)) {
-                const cssClass = key.replace('decoration', 'text');
+                let cssClass = key.replace('decoration', 'text');
+                cssClass == 'text-bf' ? cssClass = 'font-weight-bold' : cssClass == 'text-it' ? cssClass = 'font-italic' : cssClass;
```

Comment:
Hello @aab-odoo , 
I'm not very proud of these lines but I didn't find any other way to apply the fix keeping changes low... any suggestions?

Reply:
I suggest to re-introduce the two css rules :D It's 6 lines of code including the brackets

Reply:
They were defined globally, which is probably a mistake, so I don't see any problem if you decide to scope them by .o_list_view (in list_view.scss)